app/uk/gov/hmrc/partnershipidentificationfrontend/config/AppConfig.scala

```diff
@@ -16,11 +16,10 @@
 
 package uk.gov.hmrc.partnershipidentificationfrontend.config
 
+import javax.inject.{Inject, Singleton}
```

Comment:
Java and Scala standard libraries tend to be in separate list from external libraries, I think this was formatted correctly before